### ISSUE TITLE
Multiselect Placeholder text is not rendered as "placeholder" attribute and the screen readers do not announce it

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#2866]
### Bug report
Multiselect Placeholder text is not rendered as "placeholder" attribute and the screen readers do not announce it

### Reproduction of the problem
Run the following [Dojo](http://dojo.telerik.com/ohaDe/5) in IE with JAWS on Firefox with NVDA. When navigating to the <input type="text"/>, the screen reader correctly announces the placeholder. In does announce the placeholder, when navigating to the Multiselect.

**Comment by Juveniel** [comment]
The MultiSelect displays the currently selected values in a separate element - the TagList. Taking this into consideration, if the `placeholder` is set as an attribute and we have an option selected in the TagList, the placeholder text will still be displayed.  Thus, this makes the usage of the `placeholder` attribute not suitable for the current rendering of the widget. 

In order to allow the screen readers to announce the placeholder text, and also give the developers the ability to customize the placeholder text, the recommended approach is to utilize the [`change`](https://docs.telerik.com/kendo-ui/api/javascript/ui/multiselect/events/change) event of the MultiSelect as demonstrated in [the following Dojo example](https://dojo.telerik.com/aMohobew).

The above approach will be described in the MultiSelect documentation as part of #4642 and also added as a [Knowledge Base Article](https://docs.telerik.com/kendo-ui/knowledge-base).
